heutagogy/views.py

```diff
@@ -115,6 +115,20 @@ def post(self, id):
 
         return bookmark.to_dict(), HTTPStatus.OK
 
+    @token_required
+    def delete(self, id):
+        current_user_id = current_user.id
+        bookmark = db.Bookmark.query \
+                              .filter_by(id=id, user=current_user_id) \
+                              .first()
+        if bookmark is None:
+            return {'error': 'Not found'}, HTTPStatus.NOT_FOUND
+
+        db.db.session.delete(bookmark)
+        db.db.session.commit()
+
+        return {"id": bookmark.id}, HTTPStatus.OK
```

Comment:
Would be better to return `204 No Content`. That is more default response code for DELETE operation.